lib/rules/no-fibers-future-usage.js

```diff
@@ -91,7 +91,7 @@ const isUsingFibersMethods = (node) => {
 
 const includesFibersOrFuturePath = (toTest) => toTest &&
   typeof toTest === "string" &&
-  ["fibers", "future"].some((t) => toTest.toLowerCase().includes(t));
+  ["fibers", "future"].every((t) => toTest.toLowerCase().includes(t));
```

Comment:
I think this breaks from importing directly from 'fibers', isn't it?

Because it will be only `fibers` and not `future` in the string, am I wrong?

Reply:
`toTest.toLowerCase().startsWith('fibers')` would be a better check in this case?

Reply:
Yes, I think so. `fibers` is the important piece.

Reply:
Updated